src/middlewared/middlewared/alembic/versions/23.10/2023-11-10_23-19_add_exporting_table_for_reporting.py

```diff
@@ -0,0 +1,63 @@
+"""Add exporting table for reporting
+Revision ID: 8f8942557260
+Revises: 304e43883592
+Create Date: 2023-11-10 23:19:42.678757+00:00
+"""
+import json
+import sqlalchemy as sa
+
+from alembic import op
+from sqlalchemy.engine.reflection import Inspector
+from sqlalchemy.sql import text
+
+
+revision = '8f8942557260'
+down_revision = '304e43883592'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    inspector = Inspector.from_engine(conn)
+    if 'reporting_exporters' in inspector.get_table_names():
```

Comment:
Good job encountering this! Please also add this to the appropriate docs section for a specific example on how to make an idempotent migration